SU-HSDO/suhumsci
# Fix(STN-619): Remove the white space below universal footer

# READY FOR REVIEW

## Summary
There is a 1px line below the universal footer when viewing in Chrome, this is due to a 1px height being added to a Visually Hidden div that Drupal adds when you are logged into the admin area with the Toolbar menu.

![image (1)](https://user-images.githubusercontent.com/9355698/143182372-7664803f-cd26-4d61-8dd1-345044bd6601.png)

This work styles that line out, using the same techniques other Drupal large release Drupal themes use to fix this same issue.
Such as Olivero, which is the upcoming stable base Drupal theme: https://www.drupal.org/project/drupal/issues/3196976
They are applying the same styling patch to fix this.

## Need Review By (Date)
11/25

## Urgency
medium

## Steps to Test
1. npm run test
2. Check any page in Google Chrome, scroll to the bottom of the screen and make sure the 1px line is no longer showing.
3. Swap themes and make sure this applies correctly in Traditional and Colorful.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
